tests/Unit/Utilities/Test_TupleSlice.cpp

```diff
@@ -8,6 +8,26 @@
 
 #include "Utilities/TupleSlice.hpp"
 
+namespace {
+
+struct MoveTest {
+  MoveTest() = default;
+  MoveTest(const MoveTest& /*rhs*/) noexcept { status = "copy-constructed"; }
+  MoveTest& operator=(const MoveTest& /*rhs*/) noexcept {
+    status = "copied";
```

Comment:
You want to change the status in `rhs` since that's the object that is no longer "valid" after move.

Reply:
You are no longer using this status. Maybe add a comment why it's needed?